types/experiments.ts

```diff
@@ -0,0 +1,82 @@
+/**
+ * The experiment definition accessible to Firefox via Remote Settings.
+ * It is compatible with ExperimentManager.
+ */
+export interface ExperimentRecipe {
+  /** A unique identifier for the Recipe */
+  id: string;
+  /** JEXL expression defined in an Audience */
+  filter_expression: string;
+  /** Is the experiment enabled? */
+  enabled: boolean;
+  /** Experiment definition */
+  arguments: Experiment;
+}
+
+export interface Experiment {
+  /** Unique identifier for the experiment */
+  slug: string;
+  /** Publically-accesible name of the experiment */
+  userFacingName: string;
+  /** Short public description of the experiment */
+  userFacingDescription: string;
+  /** Experimenter URL */
+  experimentDocumentUrl: string;
+  /** Is the experiment currently live in production? i.e., published to remote settings? */
+  active: boolean;
+  /** Are we continuing to enroll new users into the experiment? */
+  isEnrollmentPaused: boolean;
+  /** Bucketing configuration */
+  bucketConfig: BucketConfig;
+  /** A list of features relevant to the experiment analysis */
+  features: Array<Feature>;
+  /** Branch configuration for the experiment */
+  branches: Array<Branch>;
+  /** Actual publish date of the experiment */
+  startDate: Date;
+  /** Actual end date of the experiment */
+  endDate: Date | null;
+  /** Duration of enrollment from the start date in days */
+  proposedEnrollment: number;
+  /** The slug of the reference branch */
+  referenceBranch: string | null;
+}
+
+// TODO - Needs to be generated based on Features to be added to the /data directory
+// probably like keyof typeof Features
+type Feature = string;
+
+interface BucketConfig {
+  /**
+   * The randomization unit. Note that client_id is not yet implemented.
+   * @default "normandy_id"
+   */
+  randomizationUnit: "client_id" | "normandy_id";
+  /** Additional inputs to the hashing function */
+  namespace: string;
+  /**  Index of start of the range of buckets */
+  start: number;
+  /**  Number of buckets to check */
+  count: number;
+  /**
+   * Total number of buckets
+   * @default 10000  */
+  total: number;
+}
+
+interface Branch {
+  /** Identifier for the branch */
+  slug: string;
+  /** Display name for variant, e.g. "control"  */
+  /**
+   * Relative ratio of population for the branch (e.g. if branch A=1 and branch B=3,
+   * branch A would get 25% of the population)
+   * @default 1
+   */
+  ratio: number;
+  /** Deprecated - used to indicate a type of value */
```

Comment:
Should this use `@deprecated` meta, similar to types/normandy.ts#234? 
https://github.com/mozilla/nimbus-shared/blob/f8c5a09af64ef37a794b1a2870aad7d373efb030/types/normandy.ts#L234